metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -185,6 +185,12 @@ class BigQueryV2Config(
         description="Run optimized column query to get column information. This is an experimental feature and may not work for all cases.",
     )
 
+    multiproject_lineage_support: bool = Field(
+        hidden_from_schema=True,
```

Comment:
Why is this set  - `hidden_from_schema` ? Do we not want to expose this configuration in source doc ?

Reply:
Exactly, I wanted a way to revert this change if there is any issue with it.